source/_docs/scripts.markdown

```diff
@@ -657,7 +657,10 @@ automation:
 
 {% endraw %}
 
-`conditions` also accepts a [shorthand notation of a template condition][shorthand-template].
+`conditions` also accepts a [shorthand notation of a template condition][shorthand-template]. <br>
```

Comment:
Avoid the use of HTML in Markdown.

Reply:
Sorry, I read [here](https://www.markdownguide.org/basic-syntax/#line-break-best-practices) it's actually best practice for line breaks ?

Reply:
Not using them.